src/Cake.Core/ICakeTask.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+
+namespace Cake.Core
+{
+    /// <summary>
+    /// Represents a Cake task.
+    /// </summary>
+    public interface ICakeTask : ICakeTaskInfo
```

Comment:
If anythings missing here that describes the task itself, we should probably move it to `ICakeTaskInfo`